Sources/PCCurrentUser.swift

```diff
@@ -1147,3 +1151,43 @@ public enum PCRoomMessageFetchDirection: String {
 public typealias PCErrorCompletionHandler = (Error?) -> Void
 public typealias PCRoomCompletionHandler = (PCRoom?, Error?) -> Void
 public typealias PCRoomsCompletionHandler = ([PCRoom]?, Error?) -> Void
+
+//MARK: Beams
+
+private let pushNotifications: PushNotifications = PushNotifications.shared
+
+extension PCCurrentUser {
+    /**
+     Start PushNotifications service.
+     */
+    public func enablePushNotifications() {
+        let chatkitBeamsTokenProvider = ChatkitBeamsTokenProvider(instance: self.chatkitBeamsTokenProviderInstance)
+
+        pushNotifications.start(instanceId: self.instance.id, tokenProvider: chatkitBeamsTokenProvider)
+
+        do {
+            try pushNotifications.setUserId(self.id, completion: { (error) in
```

Comment:
There should be a `pushNotifications.clearAllState()` call just in case the device was deleted server side.